specificationRepositoryConfiguration.json

```diff
@@ -24,6 +24,10 @@
       "mainRepository": "Azure/azure-sdk-for-python",
       "mainBranch": "release/v3"
     },
+    "azure-sdk-for-python-track2": {
```

Comment:
Is this just for initial testing? I thought the plan was to use the same repo but a different branch.

Reply:
By default the main branch is master, so the config is track2 targeting master, and 
Track1 targeting release/v3. That is our discussed solution for Python sdk.